{"name": "test_mul[1-0-1]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 0 == 1\n +  where 0 = <bound method Calc.mul of <test_pytest.core.calc.Calc object at 0x0000021673D27400>>(1, 0)\n +    where <bound method Calc.mul of <test_pytest.core.calc.Calc object at 0x0000021673D27400>> = <test_pytest.core.calc.Calc object at 0x0000021673D27400>.mul\n +      where <test_pytest.core.calc.Calc object at 0x0000021673D27400> = <test_pytest.tests.test_calc.TestCalc object at 0x0000021673D0D4F0>.calc", "trace": "self = <test_pytest.tests.test_calc.TestCalc object at 0x0000021673D0D4F0>, a = 1, b = 0, c = 1\n\n    @pytest.mark.parametrize('a,b,c',[\n        [1,2,2],\n        [-1,-1,1],\n        [1,-1,-1],\n        [1,0,0],\n        [1,0,1],\n        [1,-1,1]\n    ])\n    #乘法方法\n    def test_mul(self,a,b,c):\n>       assert self.calc.mul(a,b) == c\nE       assert 0 == 1\nE        +  where 0 = <bound method Calc.mul of <test_pytest.core.calc.Calc object at 0x0000021673D27400>>(1, 0)\nE        +    where <bound method Calc.mul of <test_pytest.core.calc.Calc object at 0x0000021673D27400>> = <test_pytest.core.calc.Calc object at 0x0000021673D27400>.mul\nE        +      where <test_pytest.core.calc.Calc object at 0x0000021673D27400> = <test_pytest.tests.test_calc.TestCalc object at 0x0000021673D0D4F0>.calc\n\ntest_calc.py:25: AssertionError"}, "parameters": [{"name": "a", "value": "1"}, {"name": "b", "value": "0"}, {"name": "c", "value": "1"}], "start": 1608698750379, "stop": 1608698750380, "uuid": "5f1869f1-0403-4709-b167-2ae872749a9c", "historyId": "241fb7bac75fb3bb66e1434ca5cdd510", "testCaseId": "463eb02829151a13063664521f3c4534", "fullName": "test_calc.TestCalc#test_mul", "labels": [{"name": "suite", "value": "test_calc"}, {"name": "subSuite", "value": "TestCalc"}, {"name": "host", "value": "nb2-hz20113336"}, {"name": "thread", "value": "7688-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_calc"}]}